fearless/Modules/Staking/StakingMain/View/RewardAnalyticsWidgetView.swift

```diff
@@ -0,0 +1,146 @@
+import UIKit
+import SoraUI
+import SoraFoundation
+
+final class RewardAnalyticsWidgetView: BackgroundedContentControl {
```

Comment:
Consider the same solution for selection that is used for staking widget to solve cut corner.